### ISSUE TITLE
Add FreeCAD and PrePoMax to list of interesting projects using Calculix

### ISSUE THREAD

**Issue by nicoguaro** [spolanski/CoFEA#110]
I would suggest adding the following projects

- [FreeCAD](https://www.freecadweb.org/)
- [PrePoMax](http://lace.fs.uni-mb.si/wordpress/borovinsek/?page_id=41)

**Comment by gtheler** [comment]
Is prepomax open source? I think it's not.
And it runs only on Windows so most people would not think it's even free software at all.

**Comment by nicoguaro** [comment]
I don't think that mentioning a non-opensource project substrata anything to CoFEA, does it?

BTW, the source is available to download but there isn't. License regarding its. use

**Comment by gtheler** [comment]
Well, from the CoFEA's README:

> CoFEA is an initiative that aims to bring open-source simulation software closer to the industry problems

About PrePoMax in particular, if there's no license then one has to assume it is non-free.
And since it needs an non-free OS to compile and run, it wouldn't bee free even if the source code was available.

So back to my original question on the Discourse channel. @spolanski : what is CoFEA about?

**Comment by nicoguaro** [comment]
I haven't seen the Discourse. Should I?

**Comment by gtheler** [comment]
Sure. Knock yourself out :-)

https://discord.com/channels/787682041566199818/787682041566199822/801540348018294784

**Comment by nicoguaro** [comment]
> Sure. Knock yourself out :-)
> 
> https://discord.com/channels/787682041566199818/787682041566199822/801540348018294784

The URL does not work for me.

**Comment by mdudekFEA** [comment]
Hi nicoguaron, 
thank you for opening this issue. Could you please tell me why this should be added to our benchamarks? As far as I know they are using a CalculiX as a backed FEA solver and the main changes are in wtih pre - and post-processing capabilities.

**Comment by nicoguaro** [comment]
Not in the benchmarks. There is a title called FEA Software. And under [Calculix](https://cofea.readthedocs.io/en/latest/fea_software/calculix.html) there is a title called "Interesting projects with CalculiX". I think that listing interesting projects that use CalculiX is what that section was for.

**Comment by mdudekFEA** [comment]
Yes, that's true, thank you for your suggestion! There are couple of converters which need to be added as well, so these section need a big update :)

**Comment by gtheler** [comment]
Still not open source. I liked the spirit when I did not know what it was about. I'm off.

**Comment by nicoguaro** [comment]
> Yes, that's true, thank you for your suggestion! There are couple of converters which need to be added as well, so these section need a big update :)

I added an issue and not an edit suggest because it was my first contribution.

> Still not open source. I liked the spirit when I did not know what it was about. I'm off.

You don't need to accept my suggestion. Although it seems a bit radical from your part. PrePoMax is open source, since you can access the source files. But there isn't a clear license. Have you asked the author about it?

**Comment by mdudekFEA** [comment]
Yes, he responded that PreProMax is not open source because it will add him work during development but he doesn't mind sharing the code with anyone.

**Comment by xyont** [comment]
may late to comment: [PrePoMax ](https://gitlab.com/MatejB/PrePoMax) is licensed under GPLv3, the possibility has been discussed during mid periods of development.
